### DISCUSSION TITLE
如何在手机外部浏览器使用walletconnect授权以太坊意外的链进行支付？

### DISCUSSION THREAD

**Discussion by longriver2020** [consenlabs/webview#28] [Q&A] [answered]
如题，我们的dapp希望能在外部浏览器内运行，并使用walletconnect调起imtoken，进行授权支付。但是没有找到以太坊以外的公链的支持，顺便一提，我们即将接入的是tezos链。
希望能尽快得到回复，感谢！

**Comment by unix** [comment, marked as answer]
Tezos WalletConnect 暂不支持。
